src/lnd.rs

```diff
@@ -173,6 +173,125 @@ impl LndClient {
         Ok(())
     }
 
+    /// Creates a fully populated PSBT with 1 output to the specified address of the
+    /// specified amount. Coin selection is performed automatically. After either verifying the
+    /// inputs, all input UTXOs are locked with an internal LND app ID.
+    ///
+    /// NOTE: If this method returns without an error, it is the caller's responsibility to either
+    /// spend the locked UTXOs (by finalizing and then publishing the transaction) or to
```

Comment:
Could you please help me understand this better?

So when I initiate a payjoin, and I get to the 2nd stage where I "request payjoin transaction" for the receiver to add inputs.

If they fail to add inputs, is this where these UTXOs become locked?

What commands can I use to see them, and what command to unlock them? Noting that this will be almost impossible via umbrel, so will need to have some auto unlock in the future.

Reply:
ah `release_utxos`, nice!

Reply:
This is transliteration of LND gRPC Docs to our lnd.rs docs. by locking the UTXOs, LND's guarantees the same TXO won't be accidentally doublespent.

Reply:
updating this doc to make explicit reference to `release_utxos`